chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/summarystatistics/AsynchronousSummaryStatisticsInitializor.java

```diff
@@ -45,14 +47,18 @@
 
     /**
      * Actual call to get the statistics.
-     * @param dateRange range to find statistics in
+     * 
+     * @param dateRange
+     *            range to find statistics in
      * @return a Future of type Statistics
-     * @throws InterruptedException if retrieval is interrupted
-     * @throws ExecutionException if execution fails
+     * @throws InterruptedException
```

Comment:
Could we just remove these javadocs? Like, all of them? We're not requiring them, and if the only changes to them are reformatting the EOLs I'd rather just see that change as a delete once and for all